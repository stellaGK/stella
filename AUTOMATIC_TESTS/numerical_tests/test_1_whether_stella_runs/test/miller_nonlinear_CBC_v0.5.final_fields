           # z        z-zed0           aky           akx     real(phi)     imag(phi)    real(apar)    imag(apar)    real(bpar)    imag(bpar)
  z_eqarc-zed0        kperp2
   -3.1416E+000   -3.1416E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E+000    0.0000E+000  1
   -2.6180E+000   -2.6180E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.6180E+000    0.0000E+000  1
   -2.0944E+000   -2.0944E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E+000    0.0000E+000  1
   -1.5708E+000   -1.5708E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.5708E+000    0.0000E+000  1
   -1.0472E+000   -1.0472E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E+000    0.0000E+000  1
   -5.2360E-001   -5.2360E-001    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -5.2360E-001    0.0000E+000  1
    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.4409E-016    0.0000E+000  1
    5.2360E-001    5.2360E-001    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    5.2360E-001    0.0000E+000  1
    1.0472E+000    1.0472E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E+000    0.0000E+000  1
    1.5708E+000    1.5708E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.5708E+000    0.0000E+000  1
    2.0944E+000    2.0944E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E+000    0.0000E+000  1
    2.6180E+000    2.6180E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.6180E+000    0.0000E+000  1
    3.1416E+000    3.1416E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E+000    0.0000E+000  1

   -3.1416E+000   -3.1416E+000    0.0000E+000    6.6686E-002   -1.2553E-003    1.4031E-003    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E+000    4.5922E-003  1
   -2.6180E+000   -2.6180E+000    0.0000E+000    6.6686E-002    4.8266E-004   -2.0096E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.6180E+000    4.5922E-003  1
   -2.0944E+000   -2.0944E+000    0.0000E+000    6.6686E-002   -1.2360E-004    6.1317E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E+000    4.5922E-003  1
   -1.5708E+000   -1.5708E+000    0.0000E+000    6.6686E-002    3.2324E-004   -4.8691E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.5708E+000    4.5922E-003  1
   -1.0472E+000   -1.0472E+000    0.0000E+000    6.6686E-002    1.1789E-005   -7.8033E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E+000    4.5922E-003  1
   -5.2360E-001   -5.2360E-001    0.0000E+000    6.6686E-002    9.7710E-004   -2.7814E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -5.2360E-001    4.5922E-003  1
    0.0000E+000    0.0000E+000    0.0000E+000    6.6686E-002    2.0628E-003   -5.2148E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.4409E-016    4.5922E-003  1
    5.2360E-001    5.2360E-001    0.0000E+000    6.6686E-002    2.4809E-003   -7.8761E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    5.2360E-001    4.5922E-003  1
    1.0472E+000    1.0472E+000    0.0000E+000    6.6686E-002    1.3431E-003   -5.9502E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E+000    4.5922E-003  1
    1.5708E+000    1.5708E+000    0.0000E+000    6.6686E-002    5.9666E-004   -6.0108E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.5708E+000    4.5922E-003  1
    2.0944E+000    2.0944E+000    0.0000E+000    6.6686E-002   -1.0439E-003    3.2485E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E+000    4.5922E-003  1
    2.6180E+000    2.6180E+000    0.0000E+000    6.6686E-002   -1.4935E-003   -5.1083E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.6180E+000    4.5922E-003  1
    3.1416E+000    3.1416E+000    0.0000E+000    6.6686E-002   -1.2553E-003    1.4031E-003    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E+000    4.5922E-003  1

   -3.1416E+000   -3.1416E+000    0.0000E+000   -6.6686E-002   -1.2553E-003   -1.4031E-003    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E+000    4.5922E-003  1
   -2.6180E+000   -2.6180E+000    0.0000E+000   -6.6686E-002    4.8266E-004    2.0096E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.6180E+000    4.5922E-003  1
   -2.0944E+000   -2.0944E+000    0.0000E+000   -6.6686E-002   -1.2360E-004   -6.1317E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E+000    4.5922E-003  1
   -1.5708E+000   -1.5708E+000    0.0000E+000   -6.6686E-002    3.2324E-004    4.8691E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.5708E+000    4.5922E-003  1
   -1.0472E+000   -1.0472E+000    0.0000E+000   -6.6686E-002    1.1789E-005    7.8033E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E+000    4.5922E-003  1
   -5.2360E-001   -5.2360E-001    0.0000E+000   -6.6686E-002    9.7710E-004    2.7814E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -5.2360E-001    4.5922E-003  1
    0.0000E+000    0.0000E+000    0.0000E+000   -6.6686E-002    2.0628E-003    5.2148E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.4409E-016    4.5922E-003  1
    5.2360E-001    5.2360E-001    0.0000E+000   -6.6686E-002    2.4809E-003    7.8761E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    5.2360E-001    4.5922E-003  1
    1.0472E+000    1.0472E+000    0.0000E+000   -6.6686E-002    1.3431E-003    5.9502E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E+000    4.5922E-003  1
    1.5708E+000    1.5708E+000    0.0000E+000   -6.6686E-002    5.9666E-004    6.0108E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.5708E+000    4.5922E-003  1
    2.0944E+000    2.0944E+000    0.0000E+000   -6.6686E-002   -1.0439E-003   -3.2485E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E+000    4.5922E-003  1
    2.6180E+000    2.6180E+000    0.0000E+000   -6.6686E-002   -1.4935E-003    5.1083E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.6180E+000    4.5922E-003  1
    3.1416E+000    3.1416E+000    0.0000E+000   -6.6686E-002   -1.2553E-003   -1.4031E-003    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E+000    4.5922E-003  1

   -3.1416E+000   -3.1416E+000    6.6667E-002    0.0000E+000    1.7725E-004    5.6725E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E+000    3.5424E-002  1
   -2.6180E+000   -2.6180E+000    6.6667E-002    0.0000E+000    1.1139E-004   -4.1294E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.6180E+000    2.1032E-002  1
   -2.0944E+000   -2.0944E+000    6.6667E-002    0.0000E+000    1.4684E-004   -1.4236E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E+000    1.1938E-002  1
   -1.5708E+000   -1.5708E+000    6.6667E-002    0.0000E+000   -1.9367E-004   -1.7780E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.5708E+000    7.0363E-003  1
   -1.0472E+000   -1.0472E+000    6.6667E-002    0.0000E+000   -3.1713E-004    2.4680E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E+000    4.6275E-003  1
   -5.2360E-001   -5.2360E-001    6.6667E-002    0.0000E+000   -4.6674E-004   -5.0959E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -5.2360E-001    3.5517E-003  1
    0.0000E+000    0.0000E+000    6.6667E-002    0.0000E+000   -1.0872E-004    2.6834E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.4409E-016    3.2464E-003  1
    5.2360E-001    5.2360E-001    6.6667E-002    0.0000E+000   -6.7477E-005   -4.0060E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    5.2360E-001    3.5517E-003  1
    1.0472E+000    1.0472E+000    6.6667E-002    0.0000E+000    3.3177E-004    5.9780E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E+000    4.6275E-003  1
    1.5708E+000    1.5708E+000    6.6667E-002    0.0000E+000    8.7779E-005   -2.5314E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.5708E+000    7.0363E-003  1
    2.0944E+000    2.0944E+000    6.6667E-002    0.0000E+000    2.8130E-004    1.5127E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E+000    1.1938E-002  1
    2.6180E+000    2.6180E+000    6.6667E-002    0.0000E+000    3.8899E-004   -4.2043E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.6180E+000    2.1032E-002  1
    3.1416E+000    3.1416E+000    6.6667E-002    0.0000E+000   -3.8543E-004    3.7194E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E+000    3.5424E-002  1

   -3.1416E+000   -4.3982E+000    6.6667E-002    6.6686E-002   -6.2793E-005    4.4185E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -4.3982E+000    6.2977E-002  1
   -2.6180E+000   -3.8746E+000    6.6667E-002    6.6686E-002   -3.5260E-004    1.0126E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.8746E+000    4.2049E-002  1
   -2.0944E+000   -3.3510E+000    6.6667E-002    6.6686E-002   -4.7031E-004   -4.6052E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.3510E+000    2.7440E-002  1
   -1.5708E+000   -2.8274E+000    6.6667E-002    6.6686E-002    8.3727E-006    2.5528E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.8274E+000    1.8427E-002  1
   -1.0472E+000   -2.3038E+000    6.6667E-002    6.6686E-002    7.0418E-005   -1.4970E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.3038E+000    1.3107E-002  1
   -5.2360E-001   -1.7802E+000    6.6667E-002    6.6686E-002    3.7368E-004    1.8507E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.7802E+000    9.9029E-003  1
    0.0000E+000   -1.2566E+000    6.6667E-002    6.6686E-002   -2.2460E-004   -4.0109E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.2566E+000    7.8386E-003  1
    5.2360E-001   -7.3304E-001    6.6667E-002    6.6686E-002   -4.7187E-005    1.1914E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -7.3304E-001    6.3849E-003  1
    1.0472E+000   -2.0944E-001    6.6667E-002    6.6686E-002   -5.0110E-004   -2.9470E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E-001    5.3320E-003  1
    1.5708E+000    3.1416E-001    6.6667E-002    6.6686E-002    7.0506E-004    7.8377E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E-001    4.8303E-003  1
    2.0944E+000    8.3776E-001    6.6667E-002    6.6686E-002   -5.5256E-004   -8.5756E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    8.3776E-001    5.6208E-003  1
    2.6180E+000    1.3614E+000    6.6667E-002    6.6686E-002    5.7818E-004    6.9692E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.3614E+000    9.1984E-003  1
    3.1416E+000    1.8850E+000    6.6667E-002    6.6686E-002   -3.2115E-004   -5.4379E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.8850E+000    1.7055E-002  1

   -3.1416E+000   -1.8850E+000    6.6667E-002   -6.6686E-002    5.3839E-004    1.8893E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.8850E+000    1.7055E-002  1
   -2.6180E+000   -1.3614E+000    6.6667E-002   -6.6686E-002   -1.1078E-003    1.9944E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.3614E+000    9.1984E-003  1
   -2.0944E+000   -8.3776E-001    6.6667E-002   -6.6686E-002    1.2486E-003   -2.1280E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -8.3776E-001    5.6208E-003  1
   -1.5708E+000   -3.1416E-001    6.6667E-002   -6.6686E-002   -6.8529E-004    2.5748E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E-001    4.8303E-003  1
   -1.0472E+000    2.0944E-001    6.6667E-002   -6.6686E-002    5.4673E-004    1.7287E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E-001    5.3320E-003  1
   -5.2360E-001    7.3304E-001    6.6667E-002   -6.6686E-002   -3.5016E-004   -1.3633E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    7.3304E-001    6.3849E-003  1
    0.0000E+000    1.2566E+000    6.6667E-002   -6.6686E-002    2.8420E-004   -2.6658E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.2566E+000    7.8386E-003  1
    5.2360E-001    1.7802E+000    6.6667E-002   -6.6686E-002   -2.4541E-004   -4.5687E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.7802E+000    9.9029E-003  1
    1.0472E+000    2.3038E+000    6.6667E-002   -6.6686E-002    3.0030E-004    1.5844E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.3038E+000    1.3107E-002  1
    1.5708E+000    2.8274E+000    6.6667E-002   -6.6686E-002   -1.9449E-004    2.0084E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.8274E+000    1.8427E-002  1
    2.0944E+000    3.3510E+000    6.6667E-002   -6.6686E-002    2.4050E-004   -1.4604E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.3510E+000    2.7440E-002  1
    2.6180E+000    3.8746E+000    6.6667E-002   -6.6686E-002   -1.5692E-004   -3.4193E-004    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.8746E+000    4.2049E-002  1
    3.1416E+000    4.3982E+000    6.6667E-002   -6.6686E-002    1.8948E-004   -3.3442E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.3982E+000    6.2977E-002  1

   -3.1416E+000   -3.1416E+000    1.3333E-001    0.0000E+000    6.3704E-006    1.5673E-007    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.1416E+000    1.4170E-001  1
   -2.6180E+000   -2.6180E+000    1.3333E-001    0.0000E+000    4.9457E-006    6.2227E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.6180E+000    8.4126E-002  1
   -2.0944E+000   -2.0944E+000    1.3333E-001    0.0000E+000   -3.5246E-006   -6.0546E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.0944E+000    4.7754E-002  1
   -1.5708E+000   -1.5708E+000    1.3333E-001    0.0000E+000   -3.5808E-005    3.7469E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.5708E+000    2.8145E-002  1
   -1.0472E+000   -1.0472E+000    1.3333E-001    0.0000E+000   -1.2855E-005   -1.8325E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E+000    1.8510E-002  1
   -5.2360E-001   -5.2360E-001    1.3333E-001    0.0000E+000   -6.2616E-006    1.3415E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -5.2360E-001    1.4207E-002  1
    0.0000E+000    0.0000E+000    1.3333E-001    0.0000E+000    2.0534E-005   -1.2619E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.4409E-016    1.2986E-002  1
    5.2360E-001    5.2360E-001    1.3333E-001    0.0000E+000   -1.4753E-005    4.5406E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    5.2360E-001    1.4207E-002  1
    1.0472E+000    1.0472E+000    1.3333E-001    0.0000E+000    2.4993E-005   -1.5928E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E+000    1.8510E-002  1
    1.5708E+000    1.5708E+000    1.3333E-001    0.0000E+000    1.4431E-005    6.3122E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.5708E+000    2.8145E-002  1
    2.0944E+000    2.0944E+000    1.3333E-001    0.0000E+000    9.9237E-006    5.2201E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.0944E+000    4.7754E-002  1
    2.6180E+000    2.6180E+000    1.3333E-001    0.0000E+000   -2.8159E-005    1.9286E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.6180E+000    8.4126E-002  1
    3.1416E+000    3.1416E+000    1.3333E-001    0.0000E+000    2.3845E-008    3.8296E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.1416E+000    1.4170E-001  1

   -3.1416E+000   -3.7699E+000    1.3333E-001    6.6686E-002   -1.1501E-005   -6.0923E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.7699E+000    1.9221E-001  1
   -2.6180E+000   -3.2463E+000    1.3333E-001    6.6686E-002    3.2700E-006    1.3898E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -3.2463E+000    1.2157E-001  1
   -2.0944E+000   -2.7227E+000    1.3333E-001    6.6686E-002   -3.7646E-006   -2.9205E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.7227E+000    7.4166E-002  1
   -1.5708E+000   -2.1991E+000    1.3333E-001    6.6686E-002    1.2144E-005    1.6883E-007    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.1991E+000    4.6334E-002  1
   -1.0472E+000   -1.6755E+000    1.3333E-001    6.6686E-002   -3.7019E-006   -1.6218E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.6755E+000    3.0877E-002  1
   -5.2360E-001   -1.1519E+000    1.3333E-001    6.6686E-002    2.5983E-005    1.4805E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.1519E+000    2.2317E-002  1
    0.0000E+000   -6.2832E-001    1.3333E-001    6.6686E-002   -1.2421E-005   -1.6027E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -6.2832E-001    1.7578E-002  1
    5.2360E-001   -1.0472E-001    1.3333E-001    6.6686E-002    2.9163E-005   -1.8721E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.0472E-001    1.5281E-002  1
    1.0472E+000    4.1888E-001    1.3333E-001    6.6686E-002   -9.6027E-006   -4.2786E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    4.1888E-001    1.5327E-002  1
    1.5708E+000    9.4248E-001    1.3333E-001    6.6686E-002    4.1425E-005    3.7877E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    9.4248E-001    1.9141E-002  1
    2.0944E+000    1.4661E+000    1.3333E-001    6.6686E-002    4.3142E-006    9.5319E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.4661E+000    3.0526E-002  1
    2.6180E+000    1.9897E+000    1.3333E-001    6.6686E-002    2.8697E-005    1.0920E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.9897E+000    5.5868E-002  1
    3.1416E+000    2.5133E+000    1.3333E-001    6.6686E-002    3.2333E-006    1.5976E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.5133E+000    1.0037E-001  1

   -3.1416E+000   -2.5133E+000    1.3333E-001   -6.6686E-002   -2.0208E-006   -6.8328E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -2.5133E+000    1.0037E-001  1
   -2.6180E+000   -1.9897E+000    1.3333E-001   -6.6686E-002    3.5167E-006   -3.1180E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.9897E+000    5.5868E-002  1
   -2.0944E+000   -1.4661E+000    1.3333E-001   -6.6686E-002   -4.8606E-006   -1.4300E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -1.4661E+000    3.0526E-002  1
   -1.5708E+000   -9.4248E-001    1.3333E-001   -6.6686E-002   -4.1583E-005   -7.4698E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -9.4248E-001    1.9141E-002  1
   -1.0472E+000   -4.1888E-001    1.3333E-001   -6.6686E-002   -6.2836E-005   -2.5235E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000   -4.1888E-001    1.5327E-002  1
   -5.2360E-001    1.0472E-001    1.3333E-001   -6.6686E-002   -3.4126E-005   -1.1453E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.0472E-001    1.5281E-002  1
    0.0000E+000    6.2832E-001    1.3333E-001   -6.6686E-002   -9.0235E-006   -1.5337E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    6.2832E-001    1.7578E-002  1
    5.2360E-001    1.1519E+000    1.3333E-001   -6.6686E-002    6.7142E-007   -1.1779E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.1519E+000    2.2317E-002  1
    1.0472E+000    1.6755E+000    1.3333E-001   -6.6686E-002   -9.9196E-006   -1.0738E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    1.6755E+000    3.0877E-002  1
    1.5708E+000    2.1991E+000    1.3333E-001   -6.6686E-002    5.7640E-006   -2.4828E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.1991E+000    4.6334E-002  1
    2.0944E+000    2.7227E+000    1.3333E-001   -6.6686E-002    1.5450E-007   -3.6784E-007    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    2.7227E+000    7.4166E-002  1
    2.6180E+000    3.2463E+000    1.3333E-001   -6.6686E-002   -2.1525E-005    5.6232E-006    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.2463E+000    1.2157E-001  1
    3.1416E+000    3.7699E+000    1.3333E-001   -6.6686E-002   -1.1989E-005    1.3306E-005    0.0000E+000    0.0000E+000    0.0000E+000    0.0000E+000    3.7699E+000    1.9221E-001  1

