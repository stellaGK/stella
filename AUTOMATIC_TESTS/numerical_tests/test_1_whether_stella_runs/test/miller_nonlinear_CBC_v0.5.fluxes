       #time        pflx                    vflx                    qflx
    0.0000E+000   -1.3139E-023   -8.4918E-023    2.4665E-010   -1.3450E-013   -8.3333E-023   -5.6895E-023
    1.0000E-002   -4.4906E-010   -4.4906E-010    2.3678E-010    3.7246E-010   -1.2712E-009   -1.0429E-009
    2.0000E-002   -1.6459E-010   -1.6459E-010   -2.1176E-010    2.9709E-010   -5.0773E-010   -5.9284E-010
    3.0000E-002   -2.2061E-010   -2.2061E-010   -3.7843E-010    7.6140E-013   -6.2561E-010   -7.2726E-010
    4.0000E-002   -1.1979E-010   -1.1979E-010   -1.2131E-010    1.7394E-011   -3.4315E-010   -1.4097E-010
    5.0000E-002   -7.5586E-011   -7.5586E-011   -8.7713E-011    7.6040E-012   -2.0159E-010   -2.6137E-010
    6.0000E-002   -4.0984E-011   -4.0984E-011   -1.5946E-010    2.4999E-011   -1.5763E-010   -2.3239E-010
    7.0000E-002   -4.0403E-011   -4.0403E-011    8.2721E-011    2.3546E-012   -8.3522E-011    3.9736E-011
    8.0000E-002   -4.4287E-011   -4.4287E-011   -1.6695E-010    3.1469E-011   -1.6391E-010   -1.3670E-010
    9.0000E-002   -3.0298E-011   -3.0298E-011   -6.4851E-011    5.3037E-012   -7.4387E-011   -6.0113E-011
    1.0000E-001   -3.4802E-011   -3.4802E-011   -8.3791E-011    1.5935E-011   -1.0932E-010    1.5845E-011
