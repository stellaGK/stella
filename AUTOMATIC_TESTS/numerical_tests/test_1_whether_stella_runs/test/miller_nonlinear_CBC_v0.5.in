../miller_nonlinear_CBC_v0.5.in