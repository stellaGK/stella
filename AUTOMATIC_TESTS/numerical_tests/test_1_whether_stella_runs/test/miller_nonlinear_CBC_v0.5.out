 istep=      0 time=  0.0000E+00  |phi|^2=  0.8808E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      1 time=  0.1000E-01  |phi|^2=  0.4720E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      2 time=  0.2000E-01  |phi|^2=  0.3720E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      3 time=  0.3000E-01  |phi|^2=  0.3331E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      4 time=  0.4000E-01  |phi|^2=  0.1944E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      5 time=  0.5000E-01  |phi|^2=  0.2103E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      6 time=  0.6000E-01  |phi|^2=  0.1160E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      7 time=  0.7000E-01  |phi|^2=  0.1011E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      8 time=  0.8000E-01  |phi|^2=  0.1192E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      9 time=  0.9000E-01  |phi|^2=  0.5012E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=     10 time=  0.1000E+00  |phi|^2=  0.5977E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
